cmdstanpy/stanfit/mcmc.py

```diff
@@ -301,6 +311,8 @@ def _validate_csv_files(self) -> Dict[str, Any]:
                     save_warmup=self._save_warmup,
                     thin=self._thin,
                 )
+                time_info = cast(Dict[str, float], dzero.get("time") or {})
```

Comment:
Is this just to make mypy happy? I generally prefer `# type: ignore` comments over cast statements, as casts actually do have a runtime cost